Charts: update valueErrorBar and customizeTooltip (#5065)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/valueErrorBar/valueErrorBar.md

![DevExtreme HTML5 Charts ErrorBars](/images/ChartJS/visual_elements/error_bars.png)

> Error bars can be generated either from concrete or calculated values. To generate one error bar, two values, high and low, are needed. If the [data source](/api-reference/10%20UI%20Components/BaseChart/1%20Configuration/dataSource.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource') of your chart provides concrete high and low values, assign the required data source fields to the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/valueErrorBar/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/valueErrorBar/#lowValueField) properties. To hide individual error bars, set the corresponding low and high values in the data source to `null`.

Reviewer: ```suggestion
Error bars can be generated either from concrete or calculated values. To generate one error bar, two values, high and low, are needed. If the chart [data source](/api-reference/10%20UI%20Components/BaseChart/1%20Configuration/dataSource.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource') supplies concrete high and low values, assign the required data source fields to the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/valueErrorBar/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/valueErrorBar/#lowValueField) properties. To hide individual error bars, set the corresponding low and high values in the data source to `null`.
```
